editor/js/libs/ui.three.js

```diff
@@ -669,7 +684,6 @@ class UIPoints2 extends UIPoints {
 
 		}
 
-		this.update();
```

Comment:
IMO, it's more correct to keep this line. The update callback should be executed whenever the state of the UI element changes. That is when values change.

Reply:
then we need this again ...

```js
points.onChange( null )
points.setValue( .. )
points.onChange( update )
```
... to avoid circular

Reply:
ok